presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -621,9 +649,12 @@ void PrestoServer::stop() {
   // Make sure we only go here once.
   auto shutdownOnsetSec = SystemConfig::instance()->shutdownOnsetSec();
   if (!shuttingDown_.exchange(true)) {
-    PRESTO_SHUTDOWN_LOG(INFO) << "Initiating shutdown. Will wait for "
-                              << shutdownOnsetSec << " seconds.";
-    this->setNodeState(NodeState::SHUTTING_DOWN);
+    PRESTO_SHUTDOWN_LOG(INFO) << "Shutdown has been requested. "
+                                 "Setting node state to 'shutting down'.";
+    setNodeState(NodeState::SHUTTING_DOWN);
```

Comment:
setNodeState is pretty lightweight. Shall we just merge the two logs into one? thanks!

Reply:
@xiaoxmeng 
I split it up, because there was confusion some people expressed about the previous message.
So, now we have two clear messages, so there is very little confusion.
Also, potentially, if in the future setNodeState() gets more complicated and can get stuck, we'll see it in the log.
One extra message per start/stop is not an issue.